### ISSUE TITLE
latest tag missing other arch except amd64

### ISSUE THREAD

**Issue by clarity99** [containrrr/watchtower#1837]
### Describe the bug

the latest tag on docker hub is amd64 only. this breaks all the arm servers.

### Steps to reproduce

try to run watchtower on arm server

### Expected behavior

it should run.

### Environment

- Platform ubuntu
- Architecture arm
- Docker Version

### Your logs

```text
docker compose up
[+] Running 2/1
 ✔ Container watchtower-watchtower-1                                                                                                                         Created      1.0s
 ! watchtower The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested              0.0s
Attaching to watchtower-watchtower-1
watchtower-watchtower-1  | exec /watchtower: exec format error
watchtower-watchtower-1 exited with code 0
```

**Comment by piksel** [comment]
Yes, this is due to a temporary fix to reduce the damage from v1.7.0. I will fix this properly as soon as I am able to.

**Comment by piksel** [comment]
With v1.7.1 released, the `:latest` tag is now multi-arch again. Sorry for the interruption.
